### DISCUSSION TITLE
Vehicle Handling Tab

### DISCUSSION THREAD

**Discussion by xesdoog** [YimMenu/YimMenu#2937] [Ideas]
Hello Yimmies,

First and foremost, I would like to extend my gratitude towards each and every contributor to this project. You guys rock and this thing is absolutely beautiful!

Second, I would like to have your opinion on an idea that came to my mind a few months ago:
So basically I tried to write something close to [VStancer](https://github.com/ikt32/GTAVStancer) in Lua but naturally failed like an idiot. I discarded the idea as it was clearly out of my league and moved on but today I was casually browsing through some of Yimura's GTAV classes and found this: 
https://github.com/Yimura/GTAV-Classes/blob/master/vehicle/CCarHandlingData.hpp#L51 

Look, when it comes to C++, I'm an absolute monkey but in my understanding, this felt like it could be used for an option added to the vehicle handling tab since similar... *things?* like this one:

```c++ 
float m_max_drive_bias_transfer;                //0x002C
```

are already used in YimMenu's vehicle handling. So can it be used to add another slider to adjust the camber? And if there's a similar thing for wheel offsets, add another slider to adjust them too to make wheels stick out more or fit properly *(GTA fitment is garbage especially for cars equipped with widebody kits)*. 

**Stancers** are a big thing in the GTA car community. They basically just raise/lower your suspension, modify your camber and adjust your fitment. With only the two mentioned options added, which are the main focus of stancers, YimMenu can have its own built-in stancer since 99% of the work is already done and working flawlessly with persistent profiles and everything. Just those two options are missing.

What do you think?

**Comment by NF8S** [comment]
very interested in this

**Comment by DeloCz126** [comment]
yea would be really cool
